src/it/scala/performanceanalysis/utils/TwitterFutures.scala

```diff
@@ -0,0 +1,24 @@
+package performanceanalysis.utils
+
+import com.twitter.util.{Return, Throw}
+import org.scalatest.concurrent.Futures
+
+/**
+  * Created by Ruud on 13-04-16.
+  */
+trait TwitterFutures extends Futures {
+
+  import scala.language.implicitConversions
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/scala-academy/performance-analysis/file/2403681944/issues/source?bid=3220915&fileBranchId=3232722#l11)